presto-hive/src/main/java/com/facebook/presto/hive/HiveWriteUtils.java

```diff
@@ -226,6 +234,9 @@ public static Object getField(Type type, Block block, int position)
         if (DoubleType.DOUBLE.equals(type)) {
             return type.getDouble(block, position);
         }
+        if (RealType.REAL.equals(type)) {
```

Comment:
In most cases, we have smaller to larger by class, so typically `BOOLEAN, TINYINT, SMALLINT, INTEGER, BIGINT, DECIMAL, REAL, DOUBLE, VARCHAR, VARBINARY`